data/src/main/kotlin/com/antosik/benchproject/data/popular/movies/repo/PopularMoviesDataRepository.kt

```diff
@@ -14,9 +16,21 @@ internal class PopularMoviesDataRepository(
 
     override suspend fun getPopularMoviesList(): List<MovieModel> {
         runCatching {
```

Comment:
are we catching something here ?

Reply:
No, we just avoid exceptions from fetching data from the API

Reply:
We have probably been talking about it lately. 😅

Reply:
I created issue for this #176 😄